env/index.js

```diff
@@ -10,5 +10,8 @@ loadEnv(layout.envFilePath);
 module.exports = {
     PORT: getenv.int('PORT', 9000),
     NODE_ENV: getenv('NODE_ENV', 'production'),
-    DB_URL: getenv('DB_URL', 'mongodb://localhost:27017')
+    DB_URL: getenv('DB_URL', 'mongodb://localhost:27017'),
```

Comment:
in case you want to split DB_URL, you should completely remove this variable from env